files/config.define_roots.fig

```diff
@@ -1,11 +1,13 @@
 # Define config roots (Roots are kept defined from the moment defined until all files have been parsed, they are not
 # just for current file and its sub files, but for all global includes after these have been defined
 # To make things clear, define roots as early as possible
-[%{dev,production}]
+# [@.%{dev,production}]
```

Comment:
This is I guess a misstake, should remove the comment and the @. so that it actually pre-defines the roots dev and production.